maintainers/maintainer-list.nix

```diff
@@ -1802,6 +1802,12 @@
     githubId = 442623;
     name = "Ben Pye";
   };
+  benwis = {
+    name = "Ben Wishovich";
+    email = "ben@benw.is";
+    github = "benwis";
+    githubId = 6953353;
```

Comment:
(optional) you could add your Matrix address here too, if you wanted to

Reply:
I'm not a huge matrix user, but thanks for letting me know about the option!